### ISSUE TITLE
sgtk-grid search is not working

### ISSUE THREAD

**Issue by husam212** [nwg-piotr/sway-gtk-menu#40]
If I run `sgtk-grid` the search bar doesn't work, it throws the following error on each keystroke:
```
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/sgtk_menu/grid.py", line 291, in search_items
    self.grid_favs.hide()
AttributeError: 'MainWindow' object has no attribute 'grid_favs'
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/sgtk_menu/grid.py", line 311, in search_items
    self.grid_favs.show()
AttributeError: 'MainWindow' object has no attribute 'grid_favs'
```
Running it with `-f` option works well.

**Environment:**
 - Linux distribution: Arch
 - Window Manager: Sway
 - Window Manager version: 1.4
 - Python version: 3.7.4

**Comment by nwg-piotr** [comment]
Good point. Fixed version on the way. Thank you!

**Comment by nwg-piotr** [comment]
Should be fixed in 1.2.2. AUR updated. Please close if it resolves the issue for your.
